en/admin-guide/db-customization/index.rst

```diff
@@ -0,0 +1,69 @@
+Customization of the data model
+===============================
+
+This chapter describes the possible customization to the data model.
+
```

Comment:
Please add :

"Before going further, we strongly recommend to ask the community if you have any doubts about how to store any information in the database.

If QGEP datamodel is not yet able to receive your data, please let us know what you want to store and how you would store it, QGEP model is a standard proof adaptative model. We want the standard to be as near as possible as your needs.

If none of the above solutions fit your needs, the following explanations and rules explain how to enhance QGEP for your specific needs."

Ping @sjib

Reply:
Ok. Maybe we can reference somewhere that QGEP complies with SIA 405 Waste Water and VSA-DSS datamodel of sia and VSA.

Reply:
sia: Link http://www.sia.ch/405
VSA-DSS: Link http://dss.vsa.ch

Reply:
done